docs/components/processor.rst

```diff
@@ -34,6 +34,114 @@ Then you can start and stop the processor and tweak files and all that jazz.
    The stackwalk binaries are in ``/stackwalk`` in the container.
 
 
+Processing crashes
+==================
+
+Running the processor is pretty uninteresting since it'll just sit there until
+you give it something to process.
+
+In order to process something, you first need to acquire raw crash data, put the
+data in the S3 container in the appropriate place, then you need to add the
+crash id to the "socorro.normal" RabbitMQ queue.
+
+We have helper scripts for these steps.
+
+
+``scripts/fetch_crash_data.py``
+-------------------------------
+
+This will fetch raw crash data from -prod and save it in the appropriate
+directory structure rooted at outputdir.
+
+Usage from host::
+
+  $ docker/as_me.sh scripts/fetch_crash_data.py <outputdir> <crashid> [<crashid> ...]
+
+
+For example (assumes this crash exists)::
+
+  $ docker/as_me.sh scripts/fetch_crash_data.py ./testdata 5c9cecba-75dc-435f-b9d0-289a50170818
+
```

Comment:
Why would I want to mess with the directory? Couldn't this be hardcoded to something that's also mentioned in `.gitignore`. I can appreciate that you might want to override it when you're deep in debugging but that could be `fetch_crash_data.py --save-dir=/tmp/other ...`